src/modules/commander/airspeed_calibration.cpp

```diff
@@ -109,7 +109,9 @@ int do_airspeed_calibration(orb_advert_t *mavlink_log_pub)
 		float analog_scaling = 0.0f;
 		param_get(param_find("SENS_DPRES_ANSC"), &(analog_scaling));
 		if (fabsf(analog_scaling) < 0.1f) {
-			calibration_log_critical(mavlink_log_pub, "[cal] No airspeed sensor, see http://px4.io/help/aspd");
+			calibration_log_critical(mavlink_log_pub, "[cal] No airspeed sensor, refer to the following:");
+			calibration_log_critical(mavlink_log_pub, "General: http://px4.io/docs/sensor-selection/");
+			calibration_log_critical(mavlink_log_pub, "Advanced: http://px4.io/docs/vtols-without-airspeed-sensor/");
```

Comment:
There is a limit of 50 characters for these messages, so this URL will get cropped. Can you update the patch to work with the 50 char limit? E.g. sending only the URL on the second line.